docs/fsharp/style-guide/formatting.md

```diff
@@ -646,3 +646,17 @@ type MyRecord =
 ```
 
 When applied to a parameter, they must be on the same line and separated by a `;` separator.
+
+## Formatting literals
+
+[F# literals](../language-reference/literals.md) using the `Literal` attribute should use place the attribute on its own line and camelCase naming:
```

Comment:
The sentence doesn't make sense. I think it should be "...should place the attribute on its own line and use camelCase naming:"